torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1455,6 +1455,7 @@ def _reset_lazy_init(self) -> None:
         self._streams: Dict[str, torch.cuda.Stream] = {}
         self._fsdp_graph_order: List[nn.Module] = []
         self._my_fsdp_idx_in_graph: Optional[int] = None
+        self._pre_backward_hook_full_params_prefetched: bool = False
```

Comment:
as commented in previous PR, flag is owned by its own layer only, so previous layer's flag can not be checked here

Reply:
Note that in line 2682, previous layer's _pre_backward_hook_full_params_prefetched flag is set, so in the forward of the previous layer this flag will be checked.